samples/spring-eventsourced-customer-registry/src/main/java/customer/view/CustomerByEmailView.java

```diff
@@ -1,19 +1,21 @@
 package customer.view;
 
 import customer.api.CustomerEntity;
-import customer.api.CustomerEvent;
+import customer.domain.CustomerEvent;
 import kalix.javasdk.view.View;
 import kalix.springsdk.annotations.Query;
 import kalix.springsdk.annotations.Subscribe;
 import kalix.springsdk.annotations.Table;
+import reactor.core.publisher.Flux;
+
 import org.springframework.web.bind.annotation.GetMapping;
 
```

Comment:
maybe add `@ViewId`? and if so in the rest of the views

Reply:
done